modules/mixins/match-media-base.js

```diff
@@ -54,6 +54,10 @@ var MatchMediaBase = {
   },
 
   handleMediaChange: debounce(function () {
+    if (typeof window !== 'undefined') { 
```

Comment:
Would we be better off with routing everything through `canUseDOM`? https://github.com/facebook/react/blob/master/src/vendor/core/ExecutionEnvironment.js